content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,172 @@
+
+## TL;DR
+
+`TODO(sqs): The section headers with emojis are what I think we should use, but the wording of the bullet points within are WIP`
+
+##### 🌅 Code intelligence: more useful, more times daily for every dev
+
+### A faster, simpler search experience 
+We’ve improved the search UI to make the platform more intuitive with small usability improvements while making it easier for new users to get started with tips and recommendations on the revamped homepage. Performance across the platform saw a boost by using the latest in React, lazy loading, and priority rendering. These usability and performance enhancements help you get quick answers about your codebase and get unblocked faster. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Search]
+
+Docs: provide a link to the updated docs
+
+### Understand usage and search structure with high-level aggregations of search results
+High-level aggregations of search results powered by Code Insights are now displayed in addition to results. Responses to the query are now grouped by location (repo or file), author, and arbitrary capture group pattern to help answer to questions such as:
+- Which files have the majority of these search results?
+- Which repositories is this library used in most?
+- Which library components are used most? By who?
+- What are all the possible arguments we pass to this function? 
+- And a number of [other examples](TODO ADD LINK TO SEARCH AGGREGATIONS TEMPLATES DOC – coming by late August after we test live on the feature). 
+- Aggregation can easily be reused by saving the visualization to your Code Insights dashboard.
+
+To answer these questions, you can choose to show your search results grouped by location (repo or file), author, and arbitrary capture group pattern. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Insights]
+
+[Docs](https://docs.sourcegraph.com/code_insights/explanations/search_results_aggregations)
+
+### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
+Code navigation allows you to quickly jump to symbol definitions, find references, dependencies, and more, so you can spend more time shipping code. Precise code navigation is the most powerful version of code navigation and it works cross-repository. We now support 9 languages including Ruby, Rust, Go, Java, Scala, Kotlin, Python, TypeScript, JavaScript. Auto-indexing makes precise code navigation work across all of your code, automatically, and it’s turned on by default for our Cloud customers. 
+
+<Video 
+  source={{
+    webm: 'https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/ruby-precise.webm',
+    mp4: 'https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/ruby-precise.mp4'
```

Comment:
There's a syntax error here according to [the docs](https://about.sourcegraph.com/blog/starter-pack#:~:text=Then%2C%20add%20the%20file%20path%20starting%20after%20/sourcegraph%2Dassets/%20from%20the%20Public%20URL%20of%20these%20files%20to%20the%20source%20property%20under%20mp4%20and%20webm%20respectively%2C%20without%20the%20file%20extensions.). The sources should be written as:

```suggestion
    webm: 'blog/release-post/4.0/ruby-precise',
    mp4: 'blog/release-post/4.0/ruby-precise'
```